server/resources/ProspectsResources.py

```diff
@@ -1,18 +1,24 @@
 from flask_restful import Resource, reqparse
 from models.ProspectModel import ProspectModel
+import json
 
 from flask_jwt_extended import (jwt_required, get_jwt_identity)
 
+from app import db
+
 upload_parser = reqparse.RequestParser()
-upload_parser.add_argument('prospects', type=str, location='json', required=True)
+upload_parser.add_argument('prospects', type=list, location='json', required=True)
+
+def add_owner_id(p, owner_id):
+  p['owner_id'] = owner_id
+  return p
 
 class UploadProspects(Resource):
-  #@jwt_required
+  @jwt_required
   def post(self):
-    prospects = upload_parser.parse_args().prospects
-    # TODO: insert prospects into db
-    # user_id = get_jwt_identity()
+    owner_id = get_jwt_identity()
+    prospects = [add_owner_id(p, owner_id) for p in upload_parser.parse_args().prospects]
+    print(prospects)
+    db.session.bulk_insert_mappings(ProspectModel, prospects)
+    db.session.commit()
```

Comment:
This as well, do it inside ProspectModel.py for cleanliness if possible

Reply:
I will move this to ProspectsModel though